doc/dev/developer_guide/basic-workflow.rst

```diff
@@ -221,21 +221,25 @@ as simple as::
 Automated PR validation
 -----------------------
 
-When your PR is created or updated, the Ceph project's `Continuous Integration
+When you create or update your PR, the Ceph project's `Continuous Integration
 (CI) <https://en.wikipedia.org/wiki/Continuous_integration>`_ infrastructure
-will test it automatically. At the time of this writing (September 2020), the
-automated CI testing included five tests to check that the commits in the PR
-are properly signed (see :ref:`submitting-patches`), to check that the
-documentation builds, to check that the submodules are unmodified, to check
-that the API is in order, and a :ref:`make-check` test.  Additional tests may
-be performed depending on which files are modified by your PR.
-
-The :ref:`make-check`, builds the PR and runs it through a battery of
+automatically tests it. At the time of this writing (September 2020), the
+automated CI testing included five tests: 
+
+#. a test to check that the commits are properly signed (see :ref:`submitting-patches`): 
+#. a test to check that the documentation builds
+#. a test to check that the submodules are unmodified
+#. a test to check that the API is in order
+#. a make-check test  
+   
+Additional tests may be performed depending on which files your PR modifies.
+
+The make-check builds the PR and runs it through a battery of
```

Comment:
`The make-check test builds the PR ...`

and have `make-check` again be a hyperlink

Reply:
> i.e., insert the word "test" after "make-check" and have make-check again be a hyperlink

I've finally gotten back around to this PR. Check it out.